core/src/cli_commands/prepare_update.rs

```diff
@@ -437,7 +437,7 @@ refresh_contract_parameters:
   oracle_token_id_index: 3
   min_data_points_index: 13
   min_data_points: 4
-  buffer_index: 21
+  buffer_length_index: 21
```

Comment:
Nice! I always wanted to do this but was too busy to stop what I was doing and fix it.